components/tidb_query/src/expr/builtin_string.rs

```diff
@@ -492,17 +483,13 @@ impl ScalarFunc {
         if self.field_type.get_flen() == -1
             || self.field_type.get_flen() > tidb_query_datatype::MAX_BLOB_WIDTH
         {
-            return Ok(Some(Cow::Borrowed(b"")));
+            return Ok(Some(Cow::Owned(b"".to_vec())));
         }
 
-        if let Some(size) = encoded_size(s.len()) {
-            let mut buf = vec![0; size];
-            let len_without_wrap =
-                base64::encode_config_slice(s.as_ref(), base64::STANDARD, &mut buf);
-            line_wrap(&mut buf, len_without_wrap);
-            Ok(Some(Cow::Owned(buf)))
+        if let Some(result) = expr_util::string::to_base64(s.to_vec()) {
+            Ok(Some(Cow::Owned(result)))
         } else {
-            Ok(Some(Cow::Borrowed(b"")))
+            Ok(Some(Cow::Owned(b"".to_vec())))
```

Comment:
Please help me change this to return `Ok(None)`. It's a bug I introduced before. thanks